### PR TITLE
Migrate Pipeline Runs

### PR DESCRIPTION
**Pull request by Richard87**

### CODE REVIEW

File: src/store/configs/index.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ const responseHandler: ResponseHandler = (response) => {
 
 export const costStoreApi = createApi({
   reducerPath: 'costApi',
+  keepUnusedDataFor: 24 * 60 * 60 * 1000,
```
Comment (nilsgstrabo) [inline src/store/configs/index.ts:14]: According to documentation this value is in seconds, so I guess it should be 24*60*60 if the intended value should be 24 hours.
Next question is why we want to keep the data for so long.

**Review by nilsgstrabo — CHANGES_REQUESTED** [review]
Added a few minor comments/suggestions

File: src/store/configs/index.ts (hunk 1 above)
Comment (Richard87) [reply src/store/configs/index.ts:14]: Haha, it was just an example that leaked in from the friday walk-trough, so I'll remove it asap!

But, there is an argument to keep it, in my previous job we just set it to Infinity and stored in localStorage, basicallt to immidiatly show stale data when a user visits, but also immidiatly trigger a refresh of it so they rarely see a loading icon (it does have some complications, like cache invalidation, where i used a hash of the openapi spec)

Comment (Richard87) [reply src/store/configs/index.ts]: Fjernet :)

**Review by nilsgstrabo — APPROVED** [review]

**Pull request merged**